src/App.tsx

```diff
@@ -9,7 +9,7 @@ import { AllPokemon } from "./pkmn";
 import Footer from "./Footer";
 
 function App() {
-  const [tab, changeTab] = React.useState(1);
+  var [tab, changeTab] = React.useState(2);
```

Comment:
please keep this as `const`

Reply:
ok, is it ok to change the start tab to the Pokedex, because you can now click a pokemon or a type from there?

Reply:
hmm, i think i'd like to keep it on the defense tab actually. not very many people use the dex browser:

![image](https://user-images.githubusercontent.com/176815/59460707-93935400-8dd4-11e9-9abd-6db70913fb6f.png)